src/panels/config/backup/ha-config-backup-default-config.ts

```diff
@@ -1,22 +1,107 @@
+import { customElement, property, state } from "lit/decorators";
+import { isComponentLoaded } from "../../../common/config/is_component_loaded";
+import { debounce } from "../../../common/util/debounce";
+import "../../../components/ha-button";
 import "../../../components/ha-card";
+import "../../../components/ha-expansion-panel";
+import "../../../components/ha-icon-next";
+import "../../../components/ha-list-item";
+import "../../../components/ha-md-list";
+import "../../../components/ha-md-list-item";
+import "../../../components/ha-md-select";
+import "../../../components/ha-md-select-option";
+import "../../../components/ha-password-field";
+import "../../../components/ha-select";
 import "../../../components/ha-settings-row";
 import "../../../components/ha-switch";
-import "../../../components/ha-select";
-import "../../../components/ha-button";
-import "../../../components/ha-list-item";
+import type { BackupAgent, BackupConfig } from "../../../data/backup";
+import type { BackupAddon } from "./components/ha-backup-addons-picker";
+import {
+  BackupScheduleState,
+  fetchBackupAgentsInfo,
+  fetchBackupConfig,
+  updateBackupConfig,
+} from "../../../data/backup";
+import { fetchHassioAddonsInfo } from "../../../data/hassio/addon";
+import { domainToName } from "../../../data/integration";
 import "../../../layouts/hass-subpage";
 import type { HomeAssistant } from "../../../types";
+import { brandsUrl } from "../../../util/brands-url";
+import "./components/ha-backup-addons-picker";
+import { showChangeBackupPasswordDialog } from "./dialogs/show-dialog-change-backup-password";
+
+const SELF_CREATED_ADDONS_FOLDER = "addons/local";
+
+const INITIAL_BACKUP_CONFIG: BackupConfig = {
+  create_backup: {
+    agent_ids: [],
+    include_folders: [],
+    include_database: true,
+    include_addons: [],
+    include_all_addons: true,
+    password: null,
+    name: null,
+  },
+  max_copies: 3,
+  schedule: { state: BackupScheduleState.NEVER },
+  last_automatic_backup: null,
+};
 
 @customElement("ha-config-backup-default-config")
 class HaConfigBackupDefaultConfig extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
   @property({ type: Boolean }) public narrow = false;
 
-  protected render(): TemplateResult {
+  @state() private _backupConfig: BackupConfig = INITIAL_BACKUP_CONFIG;
+
+  @state() private _agents: BackupAgent[] = [];
+
+  @state() private _addons: BackupAddon[] = [];
+
+  protected willUpdate(changedProps) {
+    super.willUpdate(changedProps);
+    if (!this.hasUpdated) {
+      this._fetchData();
+    }
+  }
+
+  private async _fetchData() {
+    if (isComponentLoaded(this.hass, "hassio")) {
+      this._fetchAddons();
+    }
+    const [backupConfig, agentInfo] = await Promise.all([
+      fetchBackupConfig(this.hass),
+      fetchBackupAgentsInfo(this.hass),
+    ]);
+    this._backupConfig = backupConfig.config || INITIAL_BACKUP_CONFIG;
```

Comment:
This will never happen because the default config returned by core is 

```json
{
    "create_backup": {
        "agent_ids": [],
        "include_addons": null,
        "include_all_addons": false,
        "include_database": true,
        "include_folders": null,
        "name": null,
        "password": null
    },
    "last_automatic_backup": null,
    "max_copies": null,
    "schedule": {
        "state": "never"
    }
}
```